src/components/ArticleList.jsx

```diff
@@ -5,27 +5,34 @@ import {useEffect, useState} from "react"
 
 export default function ArticleList () {
     const [articlesList, setArticlesList] = useState([])
+    const [loading, setLoading] = useState(true)
 
     useEffect(() => {
         axios
         .get("https://nc-news-portfolio-site.onrender.com/api/articles")
         .then((response) => {
             setArticlesList(response.data.articles)
+            setLoading(false)
         })
         .catch((err) => {
             console.log(err)
+            setLoading(false)
         })
     }, [])
 
 
     return (
+        <div>
+            {loading ? (<p>Loading...</p>) :
```

Comment:
Nice job with the loading but you will have an empty div in the DOM when loading isn't true. You can do an if statement above the return if you want to conditionally render the element. So if (true) return <p> loading<p> for example.